doc/flatpak-metadata.xml

```diff
@@ -182,6 +182,14 @@
                                 Available since 1.15.6.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>usb</option></term>
+                            <listitem><para>
+                                USB device bus
+                                (all device nodes below
+                                <filename>/dev/bus/usb</filename>).
```

Comment:
Perhaps something like 

```suggestion
                                <filename>/dev/bus/usb</filename> that are accessible
                                by the user who ran Flatpak).
```

since it's common to have USB devices that are only accessible by root (in particular keyboards and mice).

Reply:
yeah there is often confusion between "the device is node is visible" and "the device permission allow access".

Reply:
note that none of the other permission talk about it either.

Reply:
Addressed but in a more general manner.